src/index.js

```diff
@@ -18,12 +22,21 @@ const getTabValue = path => {
     return null;
 };
 
+// prettier-ignore
+console.log(`Dropbox Key = ${process.env.REACT_APP_DROPBOX_KEY}, Secret=${process.env.REACT_APP_DROPBOX_SECRET}`);
+
 const rootElement = document.getElementById('root');
 
 const Main = ({ ytAPI }) => {
     const [lastVideoId, setLastVideoId] = useState(undefined);
     const [infoText, setInfoText] = useState(undefined);
 
+    const [dropboxSetupComplete, setDropboxSetupComplete] = useState(false);
```

Comment:
Could you have something like
`const [dropboxState, setDropboxState] = useState();`

And in code, 

`if (dropboxState === 'SUCCESS') ...` and `setDropboxState('SUCCESS')` or `setDropboxState('FAILED')` ?